jdaviz/configs/imviz/helper.py

```diff
@@ -152,6 +152,40 @@ def link_data(self, **kwargs):
         """
         link_image_data(self.app, **kwargs)
 
+    def load_static_regions_from_file(self, region_file, region_format='ds9', prefix='region',
+                                      **kwargs):
+        """Load regions defined in the given file.
+        See :ref:`regions:regions_io` for supported file formats.
+
+        .. note:: Any unsupported regions will be silently skipped.
+
+        Parameters
+        ----------
+        region_file : str
+            Path to region file.
+
+        region_format : {'crtf', 'ds9', 'fits'}
+            See :meth:`regions.Regions.get_formats`.
+
+        prefix : str
+            Prefix for the Subset names generated by loaded regions.
+            Names will have the format of ``<prefix>_<i>``, where ``i``
+            is the index in the load order.
+
+        kwargs : dict
+            See :meth:`load_static_regions`.
+
+        """
+        from regions import Regions, CirclePixelRegion, RectanglePixelRegion, EllipsePixelRegion
+
+        raw_regs = Regions.read(region_file, format=region_format)
+        my_regions = {}
+        for i, reg in enumerate(raw_regs):
+            if isinstance(reg, (CirclePixelRegion, RectanglePixelRegion, EllipsePixelRegion)):
+                my_regions[f'{prefix}_{i}'] = reg
+
```

Comment:
Should we give any feedback or raise an error in the "else" case?  Does appending the index still make sense if there is only a single subregion?

Reply:
> Should we give any feedback or raise an error in the "else" case?

I debated with myself on that one. Let's say I have some old DS9 region file with, say, 50 regions. Turns out 48 of them have weird shapes that we don't support (and we don't support that many to begin with), does the user really need traceback message or 48 snackbar popups or even 1 popup that is super long? Does the user just want to see the 2 that we support anyway? 

> Does appending the index still make sense if there is only a single subregion?

Doesn't hurt to have consistent behavior whether it is one or 100 regions. But if people feel strongly about it, I can drop the index. Let's wait to hear from others?

Reply:
> Let's say I have some old DS9 region file with, say, 50 regions. Turns out 48 of them have weird shapes that we don't support (and we don't support that many to begin with), does the user really need traceback message or 48 snackbar popups or even 1 popup that is super long? Does the user just want to see the 2 that we support anyway?

One option might be to raise an error and telling the user to pass `ignore_unsupported_regions=True` (or similar)?  I agree with trying to avoid snackbars, although I guess one that says "2/50 regions successfully loaded" might work as well.

Reply:
Re: feedback -- It now returns a `dict` with regions that failed to load. So, I don't think we need errors or snackbars anymore?